### DISCUSSION TITLE
Can Kutt be used without Redis?

### DISCUSSION THREAD

**Discussion by Lenar-Avia** [thedevs-network/kutt#747] [Q&A]
Can Kutt be used without Redis? 
I.e. the performance would likely be low, 
but is it possible to start Kutt without Redis?

**Comment by poeti8** [comment]
No, but I'll make it possible in the next version.

**Reply by adblogcat** [reply]
Hi. When can we expect this version? We also wanted to remove Redis and use only Postgres

**Reply by poeti8** [reply]
Hey. Please see https://github.com/thedevs-network/kutt/issues/754. I've just started working on it. It will take several weeks at least to be finished.
